### PR TITLE
Revamped the design of Speaker Details

### PR DESCRIPTION
**Pull request by aagam-shah**

### CODE REVIEW

File: app/src/main/res/layout/activity_speaker_detail.xml [hunk 1]
```diff
@@ -1,39 +1,114 @@
...
+                        android:layout_marginStart="@dimen/default_padding"
+                        android:textColor="@color/list_item_subtext"
+                        android:textSize="@dimen/default_text_size" />
+
+                </LinearLayout>
+
+            </android.support.v7.widget.CardView>
+
+            <TextView
+                android:text="Sessions"
```
Comment (maxme) [inline app/src/main/res/layout/activity_speaker_detail.xml]: Hard coded text, should be moved to strings.xml

**Comment by maxme** [comment]
We must use a dark scrim on top of the speaker photo, to make sure the name and back button are visible. See https://www.google.com/design/spec/style/imagery.html#imagery-best-practices

Also scrolling behaves strangely on the Speaker screen, (I guess there is a problem with nested views).

**Comment by maxme** [comment]
The scrim is too small for the title and too transparent for the buttons: https://cloudup.com/cRDKJuidSBn - Scrolling is still broken.

**Comment by aagam-fk** [comment]
I tried to add more scrim, but it looks weird. Need to re-visit the design now.

**Comment by aagam-shah** [comment]
I tried to add more scrim, but it looks weird. Need to re-visit the design now.

**Comment by aagam-shah** [comment]
Closing this for now.

**Pull request closed without merging**